autopush/metrics.py

```diff
@@ -74,6 +74,14 @@ def timing(self, name, duration, **kwargs):
         self._metric.timing(name, duration)
 
 
+def make_tags(base=[], **kwargs):
```

Comment:
although this is a safe use of a list as a default kwarg, might just wanna use base=None instead (kinda surprised flake8 didn't complain, I think PyCharm will)